### PR TITLE
[update] add wstEth, dQUICK token to prices.usd

### PR DESCRIPTION
**Pull request by RaveenaBhasin**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [x] `coin_id` represents the ID of the coin on coinpaprika.com
* [x] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by RaveenaBhasin** [comment]
@jeff-dude 
How can I make it run for the arbitrum chain?
image

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 1]
```diff
@@ -92,6 +92,7 @@ FROM
     ("wbtc-wrapped-bitcoin", "arbitrum", "WBTC", "0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f", 8),
     ("weth-weth","arbitrum","WETH","0x82af49447d8a07e3bd95bd0d56f35241523fbab1",18),
     ("wis-experty-wisdom-token","arbitrum","WIS","0xa0459edcad5aac14dc32775d22ff7bd33027cac7",18),
+    ("wsteth-wrapped-liquid-staked-ether-20","arbitrum","wstETH","0x5979D7b546E38E414F7E9822514be443A4800529",18),
```
Comment (Hosuke) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:95]: API: https://api.coinpaprika.com/v1/coins/wsteth-wrapped-liquid-staked-ether-20
SCAN: https://arbiscan.io/token/0x5979D7b546E38E414F7E9822514be443A4800529

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @RaveenaBhasin

**Comment by jeff-dude** [comment]
need to resolve some issues on why the ci test is timing out, then get a successful run as expected, then we can proceed 🙏

**Comment by jeff-dude** [comment]
> @jeff-dude How can I make it run for the arbitrum chain? image

as for the arbitrum "error", it can mostly be ignored, as hosuke has proven above it's fine. when the test was built, arbitrum wasn't able to be supported. we can potentially revisit the test in the near future.

**Comment by RaveenaBhasin** [comment]
@Hosuke I've updated the PR with the dQUICK token for polygon. Kindly review it as well.

**Pull request merged**